internal/backends/winit/drag_resize_window.rs

```diff
@@ -0,0 +1,96 @@
+use winit::window::{CursorIcon, ResizeDirection};
+
+pub fn handle_cursor_move_for_resize(
+    window: &winit::window::Window,
+    position: winit::dpi::PhysicalPosition<f64>,
+    current_direction: Option<ResizeDirection>,
+) -> Option<ResizeDirection> {
+    if !window.is_decorated() {
+        let location = get_resize_direction(window.inner_size(), position, 8_f64);
```

Comment:
8 seems like a huge border.
I think it should be 1 or 2 logical pixel maximum. Don't you think?

Reply:
So I tried a few numbers and 3_f64 is the smallest I could go before I felt it started to get hard to hit it properly. Is that alright with you?